File: Data Structures & Algorithms/concatenation-of-array/submission-0.py
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        myarr = [1] * (2 *len(nums)) 
        print(myarr)
        n = len(nums)
        for i in range(len(nums)): 
            myarr[i] = nums[i] 
            myarr[i + n] = nums[i]

        return myarr 

File: Data Structures & Algorithms/duplicate-integer/submission-0.cpp
#include <set>
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        set<int> myset;
        for(int i = 0; i < nums.size(); i++){ 
            myset.insert(nums[i]);
        }
       if(myset.size() < nums.size()){ 
       //set does not allow the storage of dupe values
       //if the size is smaller then there were dupe values
       //it returns true in this case
        return true;
       }
       return false;
    }
};
